autogluon/utils/tabular/features/generators/pipeline.py

```diff
@@ -108,9 +108,9 @@ def print_feature_metadata_info(self, log_level=20):
             self._log(log_level, f'\t\tThese features do not need to be present at inference time for this FeatureGenerator.')
         if self._feature_metadata_in_unused.get_features():
             self._log(log_level, f'\tUnused Original Features (Count: {len(self._feature_metadata_in_unused.get_features())}): {self._feature_metadata_in_unused.get_features()}')
-            self._log(log_level, f'\t\tThese features were not valid input to any of the feature generators. Add a feature generator compatible with these features to utilize them.')
+            self._log(log_level, f'\t\tThese features were not used to generate any of the output features. Add a feature generator compatible with these features to utilize them.')
             self._log(log_level, f'\t\tThese features do not need to be present at inference time for this FeatureGenerator.')
```

Comment:
To reduce confusion, would replace: "These features do not need to be present at inference time for this FeatureGenerator"
with:
"These features do not need to be present at inference time."

Reply:
added